### ISSUE TITLE
Convert to pirateweather

### ISSUE THREAD

**Issue by Pokom** [shawntoffel/darksky#24]
Related to #22, is there any interest in refactoring the codebase to use https://pirateweather.net/? It's intended to be a drop in replacement for DarkSky API rest api. If you look at my fork, I've already made the necessary patches in the codebase to interface with pirateweather.

I totally understand if not, but I'll likely hard for the repo and continue to maintain it.

**Comment by shawntoffel** [comment]
Hello, thanks for submitting this issue. I'm certainly open to the idea and will take a look at your fork. In the meantime, it is already possible to change the base URL:
```go
darksky.BaseUrl = "https://api.pirateweather.net/forecast"
```

**Comment by shawntoffel** [comment]
Hello again,

I've released version [v1.6.0](https://github.com/shawntoffel/darksky/releases/tag/v1.6.0) with support for a float uVIndex in #25. The `DataPoint.UvIndex` type for this client is still int64 to avoid breaking changes. I've decided to leave the baseUrl as is for the sake of posterity, but again it can be changed with the `darksky.BaseUrl` variable. 

The float timestamp was fixed in https://github.com/alexander0042/pirateweather/issues/20 so no changes are required. 

Also, since Pirate Weather may evolve over time, I've decided to maintain a separate dedicated Go client for Pirate Weather built upon my learnings from this and the [go-weatherkit](https://github.com/shawntoffel/go-weatherkit) project. It's still in early development, but I believe it to be fully functional. 

https://github.com/shawntoffel/go-pirateweather.

Let me know if there's anything else I may have missed. Thank you.
